### ISSUE TITLE
Link helper-text to text-area with aria

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#9874]
we need to duplicate the work done in this pr https://github.com/carbon-design-system/carbon/pull/9873 for text-area as it should be having the same accessibility issue 🏄🏾
